.travis.yml

```diff
@@ -6,7 +6,7 @@ jobs:
       services:
         - docker
       env:
-        - CODALAB_USERNAME=codalab CODALAB_PASSWORD=testpassword CODALAB_ROOT_PASSWORD=testpassword CI=false CODALAB_SERVICE_HOME=/home/travis/codalab/home/ CODALAB_BUNDLE_STORE=/home/travis/codalab/bundles/ CODALAB_MYSQL_MOUNT=/home/travis/codalab/mysql/ CODALAB_WORKER_DIR=/home/travis/codalab/worker-dir/ CODALAB_MYSQL_PORT=33306 CODALAB_VERSION=${TRAVIS_BRANCH}
+        - CODALAB_USERNAME=codalab CODALAB_PASSWORD=testpassword CI=false
```

Comment:
can we have a more consistent scheme for default passwords (mysql, etc.)?  either all `codalab` or something?

Reply:
Set them all to codalab for now.